openmetadata-ui/src/main/resources/ui/src/locale/languages/en-us.json

```
@@ -1546,6 +1546,7 @@
     "hex-code-placeholder": "Select or enter HEX color code.",
     "hex-color-validation": "Input is not valid HEX code.",
     "hi-user-welcome-to": "Hi {{user}}, Welcome to",
+    "image-upload-error": "Image upload is not supported",
```

```suggestion
    "image-upload-error": "Image upload is not supported. Please use Markdown syntax for images available via URL.",
```

Or something like that? We should do this everywhere